src/components/Camera/StandardCamera.js

```diff
@@ -167,105 +174,195 @@ const StandardCamera = ( ): Node => {
     navigation.navigate( "ObsEdit" );
   };
 
-  const renderFlashButton = icon => {
+  const renderFlashButton = () => {
     let testID = "";
     let accessibilityLabel = "";
-    switch ( icon ) {
-    case "flash-on":
+    let name = "";
+    const flashClassName = ( !isTablet
+      ? `absolute bottom-[18px] left-[18px] ${cameraOptionsClassName}`
+      : `${cameraOptionsClassName} m-[12.5px]` );
+    switch ( takePhotoOptions.flash ) {
+    case "on":
+      name = "flash-on";
       testID = "flash-button-label-flash";
       accessibilityLabel = t( "Flash-button-label-flash" );
       break;
-    case "flash-off":
+    case "off":
+      name = "flash-off";
       testID = "flash-button-label-flash-off";
       accessibilityLabel = t( "Flash-button-label-flash-off" );
       break;
     default:
```

Comment:
We're not going to render the flash button if the device doesn't have a flash, right? If that's the case, I think the `off` state is the more sensible default rather then potentially rendering a button with a bunch of empty props.

There's also a bug here causing the flash button not to appear on my Pixel 3. I think it's because `const devices = useCameraDevices( "wide-angle-camera" );` is asynchronous, so when these lines run for the first time:

```javascript
const device = devices[cameraPosition];
const hasFlash = device?.hasFlash;
const initialPhotoOptions = hasFlash ? { flash: "off" } : { };
const [takePhotoOptions, setTakePhotoOptions] = useState( initialPhotoOptions );
```

`takePhotoOptions` gets initialized as an empty object literal because `devices` hasn't been populated yet.